lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -496,6 +488,18 @@ <h4><span rsf:id="msg=simplepage.layout.elements"></span></h4>
 		<ul id="error-list" rsf:id="error-list:" role="alert">
 		  <li id="error-item" rsf:id="error-item:"><span id="error-item-text" rsf:id="error-item-text">missing page</span></li>
 		</ul>
+
+		<ul class="navIntraTool">
+			<li class="bottomButtons">
+				<span id="expandCollapseButtons" style="display:none;">
+					<a href="#" role="button" id="expandAllSections" class="button menuitem"><span class="fa fa-toggle-down" aria-hidden="true" style="padding-right:5px;"></span><span rsf:id="msg=simplepage.expand.all"></span></a>
+					<a href="#" role="button" id="collapseAllSections" style="display:none;" class="button menuitem"><span class="fa fa-toggle-up" aria-hidden="true" style="padding-right:5px;"></span><span rsf:id="msg=simplepage.collapse.all"></span></a>
+				</span>
+				<a role="button" class="nextprev prev menuitem button" href="#" rsf:id="prev">next</a>
+				<a role="button" class="bottomRight nextprev prev menuitem button" href="#" rsf:id="next">next</a>
+			</li>
+		</ul>
```

Comment:
I like the move to ul/li. Should each link be wrapped in a li in this unordered list with this change?

Reply:
I agree, that it makes sense to have a list of buttons, with each button as a list-item. But I'm afraid in this case its a litte different. I reused the [structure](https://github.com/sakaiproject/sakai/blob/294fcf039f99bf6cd4d38725b49f4886770995b9/lessonbuilder/tool/src/webapp/templates/ShowPage.html#L492) used at the buttons on the bottom of the page. Here each list item has each own line. This is due to the `bottomButtons` class, which has `width: 100%` and enables the next button to actually float on the right side. Would you suggest using different styles here? I attached some screenshots now at the [Jira](https://jira.sakaiproject.org/browse/SAK-45931), to show how this looks currentlywith the PR.

Reply:
Makes sense. We can take a look at this as part of larger lessons redesign efforts.